src/components/HTMLEngineProvider/HTMLRenderers/VideoRenderer.tsx

```diff
@@ -25,6 +25,8 @@ function VideoRenderer({tnode, key}: VideoRendererProps) {
     const height = Number(htmlAttribs[CONST.ATTACHMENT_THUMBNAIL_HEIGHT_ATTRIBUTE]);
     const duration = Number(htmlAttribs[CONST.ATTACHMENT_DURATION_ATTRIBUTE]);
     const currentReportIDValue = useCurrentReportID();
+    const parentStyle = tnode.parent?.styles?.nativeTextRet ?? {};
```

Comment:
I can see this code being repeated quite a few times, can we create a util for this?